portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase

```diff
@@ -0,0 +1,45 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+    property testray.main.component.name = "Forms";
+
+    setUp {
+        TestCase.setUpPortalInstance();
+
+        User.firstLoginPG();
+
+        Navigator.openURL();
+    }
+
+    tearDown {
+        User.logoutAndLoginPG(
+            userLoginEmailAddress = "test@liferay.com",
+            userLoginFullName = "Test Test");
+    }
+
+    @description = "LRQA-82607. Access Portal Global Menu and validate CONFIGURATION string at Control Panel"
+    @priority = "5"
+    test ValidateConfigurationStringAtControlPanel {
+		ApplicationsMenuHelper.openApplicationsMenu();
+
+		ApplicationsMenu.gotoPanel(panel = "Control Panel");
+
+        AssertElementPresent(locator1 = "ApplicationsMenu#CONFIGURATION_SECTION_TITLE");
+    }
+
+    @description = "LRQA-82607. Set Screen size to custom and validate values"
+    @priority = "5"
+    test SetScreenSizeAndValidateValues {
+		MobileDevice.previewPGWithCorrectCustomNumbers(
+            height1 = 1,
+            height2 = 920,
+            width1 = 1,
+            width2 = 080);
+
+		Click.waitForMenuToggleJSClick(locator1 = "ControlMenu#SIMULATION");
+		Click.waitForMenuToggleJSClick(locator1 = "ControlMenu#SIMULATION");
+
+        MobileDevice.verifyCustomSizeWasApplied(size = "Custom Size 1920x1080 Applied");
```

Comment:
It's a good practice to have the actual assert functions at the testcase layer, instead of the macro layer. You can move the assertion from this file, and remove the macro.
> É uma boa prática ter as funções de assert na camada testcase, em vez da camada da macro. Tu pode mover a assert desse arquivo e remover da macro.